Demo/EarlGreyExample/Podfile

```diff
@@ -37,19 +34,3 @@ target "EarlGreyExampleSwiftTests" do
   pod 'EarlGrey'
```

Comment:
Add a comment stating that we don't need to add any other flag or so for Swift etc.